jgiven-junit/build.gradle

```diff
@@ -11,17 +11,9 @@ dependencies {
 
     testImplementation "junit:junit:$junitVersion"
     testImplementation "pl.pragmatists:JUnitParams:$junitParamsVersion"
-    testImplementation "org.mockito:mockito-all:1.10.19"
-    testImplementation "org.powermock:powermock-core:1.7.4"
```

Comment:
Did powermock-core become obsolete for any reason?

Reply:
Not exactly. There is a powermock-core 2.0.9, but it seems to be already imported by the modules that we use, that provide the actual API. The current configuration is also the one recommended by powermock itsself.

Hence I just tried if I can omit it, and I can :)